apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/Aggregations.cs

```diff
@@ -32,4 +32,56 @@ public static class Aggregations
     /// <returns>A count(*) aggregation.</returns>
     public static Aggregation Count(string alias = "") =>
         new Aggregation { Count = new Count(), Alias = alias };
+
+    /// <summary>
+    /// Sum of the values of the requested property.
+    ///
+    /// * Only numeric values will be aggregated. All non-numeric values
+    /// including `NULL` are skipped.
+    ///
+    /// * If the aggregated values contain `NaN`, returns `NaN`.
+    ///
+    /// * If the aggregated value set is empty, returns 0.
+    ///
+    /// * Returns a 64-bit integer if the sum result is an integer value and does
+    /// not overflow. Otherwise, the result is returned as a double. Note that
+    /// even if all the aggregated values are integers, the result is returned
+    /// as a double if it cannot fit within a 64-bit signed integer. When this
+    /// occurs, the returned value will lose precision.
+    ///
+    /// * When underflow occurs, floating-point aggregation is non-deterministic.
+    /// This means that running the same query repeatedly without any changes to
+    /// the underlying values could produce slightly different results each
+    /// time. In those cases, values should be stored as integers over
+    /// floating-point numbers.
+    /// </summary>
+    /// <param name="property">Property for which Sum is to calculated</param>
+    /// <param name="alias">A string used to retrieve the result of this aggregation.
+    /// If not provided, Datastore will pick a default name following the format `property_&lt;incremental_id++>`
+    /// eg. property_1
+    /// </param>
+    /// <returns>A Sum(property) aggregation.</returns>
+    public static Aggregation Sum(string property, string alias = "") =>
+        new() { Sum = new Sum() { Property = new PropertyReference(property) }, Alias = alias };
```

Comment:
Might as well use target-typed new in both places:

```csharp
new() { Sum = new() { Property = new PropertyReference(property) }, Alias = alias };
```

Or if you *do* want to keep `new Sum`, remove the `()` after it to be consistent with our style for object initializers.